app/app.py

```diff
@@ -28,6 +29,10 @@ def handle_completed(self, subproblem: Subproblem):
     def all_subproblems_finished(self) -> bool:
         return not (self.pool.not_started_pool or self.pool.in_progress_pool)
 
+    def broadcast_result(self, subproblem: Subproblem, broker: Broker):
```

Comment:
Move this operation to `__main__.py`. It has nothing to do with ComputationManager